perfectTestWishList/src/main/java/pages/Header.java

```diff
@@ -24,4 +25,7 @@ public WishListPage goToWishList(){
         click(buttonWishList);
         return new WishListPage(driver);
     }
+
+
+
```

Comment:
Darynka Penkova,
Could you please delete these lines cause they are useless

Reply:
Orest yhodi